### PR TITLE
replace OlapMeta with KVStore

### LINKED ISSUE #74: Rename OlapMeta to KVStore
https://github.com/StarRocks/starrocks/blob/dbb993b476f3964ea8d06811b36716e6b5b3c0ff/be/src/storage/olap_meta.h#L45
The class `OlapMeta` is a thin wrapper on RocksDB to provides key-value storage interfaces, the `OlapMeta` is somehow inaccurate, suggest renaming it to `KVStore`.

### PR DESCRIPTION
**Pull request by Narcasserun**
1.replace OlapMeta with KVStore

### CODE REVIEW

**Comment by Astralidea** [comment]
Hi @Narcasserun you could run 
```
cd build-support/
sh clang-format.sh
```
for format BE code to pass starrocks_clang-format test.

**Comment by Narcasserun** [comment]
> Hi @Narcasserun you could run
> 
> ```
> cd build-support/
> sh clang-format.sh
> ```
> 
> for format BE code to pass starrocks_clang-format test.

I tried, but it suggested： which: no clang-format in，Can you help me

**Comment by Astralidea** [comment]
if you OS is mac you can setup a brew from https://brew.sh/ and then run 
```
brew info clang-format
```
and if you OS is ubuntu you just run
```
sudo apt-get install clang-format
```
to setup clang-format

File: be/src/storage/olap_meta.cpp [hunk 1]
```diff
@@ -51,9 +51,9 @@ const std::string META_POSTFIX = "/meta"; // NOLINT
 const std::string SECOND_POSTFIX = "_secondary";
 const size_t PREFIX_LENGTH = 4;
 
-OlapMeta::OlapMeta(std::string root_path) : _root_path(std::move(root_path)), _db(nullptr) {}
+KVStore::KVStore(std::string root_path) : _root_path(std::move(root_path)), _db(nullptr) {}
```
Comment (sduzh) [inline be/src/storage/olap_meta.cpp:54]: You can rename `olap_meta.h/cpp` to `kv_store.h/cpp`.

Comment (Narcasserun) [reply be/src/storage/olap_meta.cpp:54]: ok,thanks

**Pull request closed without merging**